### PR TITLE
Added icons for some languages and folders :)

### PR DESCRIPTION
**Pull request by itsamedood**
# Includes:

## Files:
- `.lua`
- `.hx` & `.hxml`
- `.spec` (ruby)
- `.xml`
- `.bf`
- `.editorconfig`
- `.pkg`

## Folders:
- `commands`
- `events`
- `bin` & `out`

Also updated `tasks.json`; the command run now creates `/file-icons` `/product-icons` before running the python code.
Also added `__pycache__` to `.gitignore`.

This repo is cool, love the idea of emojis for icons :)

### CODE REVIEW

**Comment by mightbesimon** [comment]
thank you for your contribution and happy New Year 😎
I'll add your name in `README.md` before the next release

File: .vscode/tasks.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		{
 			"label": "export",
 			"type": "shell",
```
Comment (mightbesimon) [inline .vscode/tasks.json:8]: btw you can simplified this to `mkdir -p file-icons product-icons`

Comment (mightbesimon) [reply .vscode/tasks.json:8]: resolved in https://github.com/mightbesimon/vscode-emoji-icons/commit/89b804e097c4e20f6f5d521b8a1b109c5977694a

File: .vscode/tasks.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
-			"command": "python3 -m translate",
+			"command": "mkdir -p file-icons; mkdir -p product-icons; python3 -m translate",
```
Comment (mightbesimon) [inline .vscode/tasks.json:9]: btw you can simplified this to `mkdir -p file-icons product-icons` in one line
resolved in https://github.com/mightbesimon/vscode-emoji-icons/commit/89b804e097c4e20f6f5d521b8a1b109c5977694a

File: references/extension-icons.md [hunk 3]
```diff
@@ -63,6 +71,7 @@ development
 - 🤐 .zip | .tar.gz
 - 🚀 .yml | .yaml
 - ⚙️ .env | .cfg | .config
+- 🐀 .editorconfig
```
Comment (mightbesimon) [inline references/extension-icons.md:74]: this would be in `references/file-icons.md` for special files

**Review by mightbesimon — COMMENTED** [review]
thank you for your contribution and happy New Year 😎
I really appreciate the lua and bf extensions and thank you for fixing my build task 😊
I've added your name in `README.md` contributors (as `dood`) for the next release

**Comment by itsamedood** [comment]
Thanks man! Happy new year to you too, thanks for this extension, it's simple yet kinda cool! Thanks for fixing my mistakes, I guess I suck at reading lol, and for pointing out `mkdir` can take more than one folder name at a time.

heck-yeah-bro

**Pull request merged**